src/compiler/scala/tools/nsc/transform/UnCurry.scala

```diff
@@ -247,8 +247,9 @@ abstract class UnCurry extends InfoTransform
         }
       }
 
-    def transformArgs(pos: Position, fun: Symbol, args: List[Tree], formals: List[Type]): List[Tree] = {
+    def transformArgs(pos: Position, fun: Symbol, args: List[Tree], params: List[Symbol]): List[Tree] = {
       val isJava = fun.isJavaDefined
+
```

Comment:
As a matter of cleanup, could we add the result type of `transformVarArgs` ? I really get confused with methods that do not fit a screenshot.